docker/docker-compose.yml

```diff
@@ -1,7 +1,19 @@
 services:
   ghostfolio:
-    image: ghostfolio/ghostfolio:latest
+    image: docker.io/ghostfolio/ghostfolio/ghostfolio:latest
     init: true
+    read_only: true
+    cap_drop:
+      - ALL
+    security_opt:
+      - no-new-privileges:true
+    # uncomment to set resource usage limits
```

Comment:
Can you remove the resource usage limits? Anyone who needs it can configure it themselves.

Reply:
I've added some examples on resource usage limits because it's recommended to set them in order to prevent DoS attacks (see https://cheatsheetseries.owasp.org/cheatsheets/Docker_Security_Cheat_Sheet.html#rule-7-limit-resources-memory-cpu-file-descriptors-processes-restarts), these can be difficult to adjust because of different usage levels of the services so I left them commented out. Happy to remove them if you think it doesn't add value

Reply:
Yes, I would leave it out for the sake of simplicity. Anyone can tweak this file as they wish.